### ISSUE TITLE
Container image configures the MPP to use IP address

### ISSUE THREAD

**Issue by alexevansigg** [HewlettPackard/csa-ce#2]
CSA shouldnt need the IP address of the container for the installation... If we specify a Hostname that should be enough. 

Deploying this docker container as is described, makes a configuration in which the mpp login always redirect to provied IP Address... which in many dev setups would not be a static IP.

**Comment by mbettan** [comment]
I always agree with Alex. I think they require fqdn hostname and IP address for the installation setup, but it would be better to workaround it. Is it for static redirect mpp?

**Comment by alexevansigg** [comment]
Yeah IPAddress is used for the network interface config on the container, but the CSA APP in the container need to redirect to the Hostname/Domain name provided.

**Comment by PO3** [comment]
Hi, 
it seems that the IP address is used for LOGIN redirection while hitting the LOGIN button from the market place page : https://csa.contoso.com:18089/org/CSA_CONSUMER. It redirects to
https://10.30.10.4:18444/idm-service/idm/v0/login?tenant=CSA_CONSUMER&token=7bV...
It produces following problems:
- If the provided IP address is internal while accessing from outside (and vice versa), the login won't work. While using IP addresses, it must be documented what purpose is that IP and if it must be internal or external. On my environment, I just cannot log to the marketplace.
- It is used with an SSL certificate, so moving IP may be difficult. How about load balancer?
This IP address should be removed from the configuration.
Thanks, PO3
